.buildkite/nightly.yml

```diff
@@ -9,7 +9,6 @@ env:
   CABAL_DIR: "/build/cardano-wallet.cabal"
   XDG_STATE_HOME: "/build/cardano-wallet/.state"
   XDG_CACHE_HOME: "/build/cardano-wallet/.cache"
-  TESTS_LOGDIR: "/build/cardano-wallet/integration-test-logs"
```

Comment:
Is there a reason not to remove other env vars too, given that in other pipelines they are removed?